.changeset/regex-locator.md

```diff
@@ -0,0 +1,5 @@
+---
+@interactors/html: patch
+---
+
+Allow use a regular expression for searching element
```

Comment:
```suggestion
Allow use a regular expression (in addition to locator string and filter) as a primary method for looking up an interactor. E.g. `Button(/submit/i)`
```

Reply:
I'll create an issue for it and update the docs this week